test/line/length_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'minitest/autorun'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.